src/styles/_dashboard.scss

```
@@ -106,16 +106,17 @@
       }
 
       .languages {
-        color: rgba(255, 255, 255, 0.7);
-        font-size: 2rem;
-
         span {
+          background: rgba(255, 255, 255, 0.1);
           margin: 0 1rem 0 0;
+          padding: 0.5rem 0.8rem;
+          border-radius: 10% 10% 10% 10%;
```

An easier way of writing this when the radius of all sides are same. 

```suggestion
          border-radius: 10%;
```